docs/user-guide/projects/project-settings/files/README.md

```diff
@@ -9,25 +9,29 @@ keywords: file storages, project folder, storages
 
 > **Info**: Before a storage service can be added to a project, an administrator must first set up [Nextcloud integration](../../../../system-admin-guide/integrations/nextcloud/) or [OneDrive/SharePoint integration](../../../../system-admin-guide/integrations/one-drive/) with OpenProject.
 
-| Topic                                                                                                            | Description                                       |
-|------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------|
-| [Activate File Storages module](#activate-file-storages-module)                                                  | Activate File Storages module in a project        |
-| [Add Nextcloud storage](#add-a-nextcloud-storage-to-a-project)                                                   | Add your Nextcloud storage to a project           |
-| [Nextcloud and project folders](#project-folders)                                                                | How to manage Nextcloud project folders           |
-| [Edit and delete Nextcloud storage](#edit-an-existing-nextcloud-storage)                                         | Edit and delete an existing Nextcloud storage     |
+| Topic                                                        | Description                                       |
+| ------------------------------------------------------------ | :------------------------------------------------ |
+| [Files module overview](#files-module-overview)              | Files module explained                            |
+| [Add Nextcloud storage](#add-a-nextcloud-storage-to-a-project) | Add your Nextcloud storage to a project           |
+| [Nextcloud and project folders](#project-folders)            | How to manage Nextcloud project folders           |
+| [Edit and delete Nextcloud storage](#edit-an-existing-nextcloud-storage) | Edit and delete an existing Nextcloud storage     |
 | [Add OneDrive/SharePoint storage to a project](#add-a-onedrivesharepoint-storage-to-a-project-enterprise-add-on) | Add your OneDrive/SharePoint storage to a project |
 
-## Activate File Storages module
+## Files module overview
 
-Once you are certain that an administrator has set up a file storage integration (Nextcloud or OneDrive/SharePoint), navigate to any existing project in your OpenProject instance and click on **Project settings -> Modules**. There, activate the **File Storages** module by clicking on the checkbox next to it and then clicking on **Save**.
+Under the **Files** module you will be able to manage external file storages, such as Nextcloud or OneDrive/SharePoint and manage direct attachments upload under *Files* tab in work packages. Navigate to **Project settings -> Files** to get here. 
```

Comment:
@MayaBerd is the term direct attachment something we are using? I never heard of it referred as direct attache, rather internal attachments or attachments. If it is an agreed term ignore this comment.

Reply:
@marcalcobe no, I think it's just a term I used based my personal understanding. It might make sense to at least partially remove the word *direct*